pkg/skaffold/schema/validation/validation.go

```diff
@@ -98,22 +97,31 @@ func validateTaggingPolicy(bc latest.BuildConfig) (errs []error) {
 	return
 }
 
-// validateImageNames makes sure the artifact image names are valid base names,
+// validateImageNames makes sure the artifact image names are unique and valid base names,
 // without tags nor digests.
-func validateImageNames(artifacts []*latest.Artifact) (errs []error) {
-	for _, a := range artifacts {
-		parsed, err := docker.ParseReference(a.ImageName)
-		if err != nil {
-			errs = append(errs, fmt.Errorf("invalid imageName '%s': %v", a.ImageName, err))
-			continue
-		}
+func validateImageNames(configs []*latest.SkaffoldConfig) (errs []error) {
```

Comment:
Aside: we should define a `SkaffoldConfigs` type that is `[]*latest.SkaffoldConfig` (doesn't need to be done here)